base/plugins/score-plugin-engine/Engine/Protocols/Joystick/JoystickDevice.cpp

```diff
@@ -0,0 +1,60 @@
+
+#include "JoystickDevice.hpp"
+#include "JoystickSpecificSettings.hpp"
+#include <ossia/network/joystick/joystick_protocol.hpp>
+
+
+W_OBJECT_IMPL(Engine::Network::JoystickDevice)
+
+namespace Engine {
+
+    namespace Network {
```

Comment:
use clang-format with the .clang-format file in the repo on these files

Reply:
I will do that I have forgottent !

Reply:
done